hotel-management/src/pages/HomePage/index.tsx

```diff
@@ -46,20 +22,15 @@ export const HomePage = () => {
           </Button>
         </Link>
       </div>
-      <ul className="list-room">
-        {testRooms.map(({ id, name, price, isAvailable }) => (
-          <RoomCard
-            key={id}
-            roomId={id}
-            name={name}
-            price={price}
-            isAvailable={isAvailable}
-            src=""
-            onDelete={() => {}}
-            onEdit={() => {}}
-          />
-        ))}
-      </ul>
+      <div className="list-room-container">
+        {isLoading ? (
+          <Spinner size="lg" />
+        ) : error ? (
+          <Message content={{ FETCH_ROOMS_FAILED }} />
+        ) : (
+          <RoomList rooms={rooms} />
+        )}
```

Comment:
Split into a function to render using switch case or if. This one is hard to read